collector/container/konflux.Dockerfile

```diff
@@ -197,5 +173,3 @@ LABEL \
     com.redhat.component="rhacs-collector-container" \
     io.k8s.display-name="collector" \
     name="rhacs-collector-rhel8"
```

Comment:
Technically, these labels don't control anything in Konflux. They are informative (and are checked by EC) but nothing more. If we're to get rid of `collector-slim` at some point, I would suggest you can drop `FROM collector-common AS collector-slim` stage altogether and set `build-target-stage` to empty in `.tekton/collector-slim*.yaml` files.

This could be done later as well.